docker/fedora/setup-and-start.sh

```diff
@@ -7,4 +7,25 @@ if [ ! -f /opt/fedora/tomcat/bin/catalina.sh ]; then
   echo 'Done running installer jar...'
 fi
 
+echo "Manually unpacking WAR to override libraries"
+mkdir -p /opt/fedora/webapp-tmp/fedora
+mv /opt/fedora/tomcat/webapps/fedora.war /opt/fedora/webapp-tmp/fedora/
+cd /opt/fedora/webapp-tmp/fedora
+jar -xvf fedora.war
+cd /opt
+mv /opt/fedora/webapp-tmp/fedora /opt/fedora/tomcat/webapps/
+
+rm /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/httpclient-4*.jar
+rm /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/httpcore-4*.jar
+rm /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/httpclient-4*.jar
+cp /opt/jars/apache-http/*.jar /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/
+rm /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/s3-url-protocol-*.jar
+cp /opt/jars/cul/s3-url-protocol-0.1.jar /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/
+cp /opt/jars/awssdk/*.jar /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/
+rm /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/fcrepo3-s3-server-*.jar
+cp /opt/jars/cul/fcrepo3-s3-server-0.1.jar /opt/fedora/tomcat/webapps/fedora/WEB-INF/lib/
+echo "Done overriding Fedora 3 libraries; setting new FCFG config"
+cp /opt/fedora.delegating-external.fcfg /opt/fedora/server/config/fedora.fcfg
+cp /opt/permit-all-s3-resolution.xml /opt/fedora/data/fedora-xacml-policies/repository-policies/default/permit-all-s3-resolution.xml
```

Comment:
It turns out that there is no /opt/fedora/data/ directory available when this script runs, so it looks like this `cp` operation is silently failing.

Reply:
I fixed this in the branch that I merged into this one, so I'll mark this comment as resolved.